src/azure-cli/azure/cli/command_modules/appconfig/keyvalue.py

```diff
@@ -311,17 +320,29 @@ def export_config(cmd,
     elif destination == 'appservice':
         dest_kvs = __read_kv_from_app_service(cmd, appservice_account=appservice_account)
 
-    need_kv_change = __print_kv_preview(
-        old_json=__serialize_kv_list_to_comparable_json_object(keyvalues=dest_kvs, level=destination),
-        new_json=__serialize_kv_list_to_comparable_json_object(keyvalues=src_kvs, level=destination),
-        yes=yes)
+    kv_comparer = KVComparer(
+        src_kvs=src_kvs,
+        compare_fields=CompareFieldsMap[destination],
+        preserve_labels=destination == "appconfig" and preserve_labels,
+        label=dest_label)
+
```

Comment:
do we need to pass content type here?

Reply:
The content type is not needed here as we do not specify destination content type for key-value export.